eslint/babel-eslint-parser/src/parse.js

```diff
@@ -26,15 +26,15 @@ export default function(code, options) {
     ignore: options.babelOptions.ignore,
     only: options.babelOptions.only,
     parserOpts: {
-      allowImportExportEverywhere: options.allowImportExportEverywhere, // consistent with espree
```

Comment:
This option was introduced at https://github.com/babel/babel-eslint/pull/329 and intended to pass through to `@babel/parser`, so the comment here might be a little bit confusing. You can change it to `// consistent with babel-parser`.

Therefore, I would say we leave it as-is, since there is no strong reason that we should drop this option.

Reply:
Ah okay, that makes sense. Thanks!